core/src/main/resources/hudson/model/CauseAction/summary.jelly

```diff
@@ -24,14 +24,32 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <t:summary icon="orange-square.png">
-    <j:forEach var="entry" items="${it.causeCounts.entrySet()}">
-      <p>
-        <st:include page="description.jelly" it="${entry.key}"/>
-        <j:if test="${entry.value > 1}">
-          <st:nbsp/>${%Ntimes(entry.value)}
-        </j:if>
-      </p>
-    </j:forEach>
-  </t:summary>
+	<t:summary icon="orange-square.png">
+	<h4>${%Cause}</h4>
+	<style type="text/css">
```

Comment:
I suspect it impacts entire page, not only the summary block. Maybe requires more unique names